nette/notejam/app/presenters/templates/Note/delete.latte

```diff
@@ -0,0 +1,9 @@
+{block title}{$note->name}{/block}
+{block header}{$note->name}{/block}
+{block content}
+	<p>Are you sure you want to delete the note?</p>
+	{form deleteNoteForm}
```

Comment:
I agree that it's good to show options, but it should also be clear which ones are better for what :)